etc/kayobe/kolla/globals.yml

```diff
@@ -103,5 +103,8 @@ prometheus_ceph_mgr_exporter_endpoints:
 {% endfor %}
 {% endif %}
 
+# Use inventory hostnames as labels
+prometheus_instance_label: "{{ ansible_facts.hostname }}"
```

Comment:
Needs to be wrapped in `{% raw %}`

Reply:
Also this should be targeted at Zed because it's a breaking change.

Reply:
or it's available in yoga but we don't want it to be the default?

Reply:
It's a breaking change, so we don't want to apply it by default on yoga. Individual deployments can still apply it on yoga if they wish.